app/src/ui/branches/branches-container.tsx

```diff
@@ -181,9 +184,7 @@ export class BranchesContainer extends React.Component<
           pullRequests={pullRequests}
           onSelectionChanged={this.onPullRequestSelectionChanged}
           filterText={this.state.pullRequestFilterText}
-          onFilterKeyDown={this.closeFoldoutOnEsc(
-            () => this.state.pullRequestFilterText.length === 0
-          )}
+          onFilterKeyDown={this.onPullRequestFilterKeyDown()}
```

Comment:
`{this.onPullRequestFilterKeyDown}`?

Reply:
oh, I see how you’re using this. That still ends up creating a new function in every render.

Reply:
Suggestions to prevent this? I know I can duplicate the code, but was hoping to avoid that.

Reply:
You could do `onPullRequestFilterKeyDown = this.closeFoldoutOnEsc(...)` in the class body instead of defining it as a method.

Reply:
Ahh, so like a field. Seems simple enough 👍

Reply:
Linter wouldn't allow me to do that, so I just duplicated code.